### ISSUE TITLE
RTMDet different loss than official paper ?

### ISSUE THREAD

**Issue by PushpakBhoge** [open-mmlab/mmdetection#11732]
The official paper of RTM Det https://arxiv.org/pdf/2212.07784 link the RTMDet of mmdet as official implementation of the paper but there is one weird thing I have noticed that official paper mentions that the bbox_loss is -log(IOU)

but in config it using GIOU loss ? I thought maybe in code this log was being taken but I did not find any such operations in class `RTMDetHead` method ?

is there any reason why this part is different from the paper ?
Also why have GIoU loss and not CIoU loss

tagging authors here thank you for your help
@RangiLyu  @ZwwWayne

**Comment by tinggh** [comment]
The cls loss is quality focal loss just as the gfl paper, but not as the offical paper. Also the region cost is not uesed inthe offical code.

**Comment by tinggh** [comment]
> The cls loss is quality focal loss just as the gfl paper, but not as the offical paper. Also the region cost is not uesed inthe offical code.

oh, I know the loss in the offical paper is just used as the cost in dynamic soft label assigner, the head cls loss and box loss is just as the quality focal loss and giou loss.

**Comment by PushpakBhoge512** [comment]
yes you are right I did find this code in the assigned class Thank you for the help @tinggh

**Comment by PushpakBhoge** [comment]
yes you are right I found that piece of code in the assigned class Thank you for your help @tinggh
